src/settings/backendsettingspage.ui

```diff
@@ -132,7 +132,7 @@
          <item>
           <spacer name="spacer_alsaplugin">
            <property name="orientation">
-            <enum>Qt::Horizontal</enum>
+            <enum>Qt::Orientation::Horizontal</enum>
```

Comment:
These changes shouldn't be part of the PR since it's unrelated to the network remote.

Reply:
Agree, but not sure how it got there or how to resolve this